contrib/gmsh-node-tuples.py

```diff
@@ -0,0 +1,161 @@
+import gmsh
+
+
+OUTPUT_TEMPLATE = """# GENERATED by gmsh_interop/contrib/gmsh-node-tuples.py
+# GMSH_VERSION: %s
+# DO NOT EDIT
+
+triangle_data = %s
+
+tetrahedron_data = %s
+
+quadrangle_data = %s
+
+hexahedron_data = %s
+"""
+
+
+TRIANGLE_ELEMENTS = {
+        "MSH_TRI_3": 2,
+        "MSH_TRI_6": 9,
+        "MSH_TRI_10": 21,
+        "MSH_TRI_15": 23,
+        "MSH_TRI_21": 25,
+        "MSH_TRI_28": 42,
+        "MSH_TRI_36": 43,
+        "MSH_TRI_45": 44,
+        "MSH_TRI_55": 45,
+        "MSH_TRI_66": 46,
+        }
+
+
+TETRAHEDRON_ELEMENTS = {
+        "MSH_TET_4": 4,
+        "MSH_TET_10": 11,
+        "MSH_TET_20": 29,
+        "MSH_TET_35": 30,
+        "MSH_TET_56": 31,
+        "MSH_TET_84": 71,
+        "MSH_TET_120": 72,
+        "MSH_TET_165": 73,
+        "MSH_TET_220": 74,
+        "MSH_TET_286": 75,
+        }
+
+
+QUADRANGLE_ELEMENTS = {
+        "MSH_QUA_4": 3,
+        "MSH_QUA_9": 10,
+        "MSH_QUA_16": 36,
+        "MSH_QUA_25": 37,
+        "MSH_QUA_36": 38,
+        "MSH_QUA_49": 47,
+        "MSH_QUA_64": 48,
+        "MSH_QUA_81": 49,
+        "MSH_QUA_100": 50,
+        "MSH_QUA_121": 51,
+        }
+
+
+HEXAHEHEDRON_ELEMENTS = {
+        "MSH_HEX_8": 5,
+        "MSH_HEX_27": 12,
+        "MSH_HEX_64": 92,
+        "MSH_HEX_125": 93,
+        "MSH_HEX_216": 94,
+        "MSH_HEX_343": 95,
+        "MSH_HEX_512": 96,
+        "MSH_HEX_729": 97,
+        "MSH_HEX_1000": 98,
+        }
```

Comment:
It looks like this information (the element type IDs) is maintained redundantly here and in `gmsh_interop/reader.py`. Could you add that to the generated bits so we don't have to have it twice?

Reply:
Fixed in 7e53562

Added an `element_type` to each class and used that to generate the `gmsh_element_type_to_info_map`.